src/KafkaFlow.Retry.SqlServer/Deploy/01 - Create_Tables.sql

```diff
@@ -130,6 +130,41 @@ BEGIN
 END
 GO
 
+IF(NOT EXISTS(SELECT *
+              FROM sys.types
+              WHERE is_table_type=1 AND name='TY_RetryQueueItemsIds'))
+BEGIN
+    CREATE TYPE [dbo].[TY_RetryQueueItemsIds]
+    AS 
+    TABLE
+    (
+     Id BIGINT
+    );
+END;
+GO
+
+IF(EXISTS(SELECT *
+          FROM sys.objects
+          WHERE type='P' AND OBJECT_ID=OBJECT_ID('[dbo].[P_LoadItemMessages]')))
+BEGIN
+    DROP PROCEDURE [dbo].[P_LoadItemMessages];
+END;
+GO
+
+CREATE PROCEDURE [dbo].[P_LoadItemMessages]
```

Comment:
Actually, the common naming conventions used across the SQL Server community define names starting with "SP_", as in "Stored Procedure". It's a small detail, not that really important, if you want, change it in the next release, otherwise, just leave this as is.

Reply:
I think that all master database stored procedures start with `SP_`. Maybe we should use a different approach to differentiate our own implementations. My suggestion is to use something like `spLoadItemMessages` 
![image](https://user-images.githubusercontent.com/29845919/201401281-23dcca04-d9cb-4450-8478-0ad59dfdcf2e.png)